ext/http/01_http.js

```diff
@@ -297,6 +297,13 @@
 
           ws[_eventLoop]();
         }
+      } else {
+        // Try to close "request" resource. It might have been already consumed,
+        // but if it hasn't been we need to close it here to avoid resource leak.
+        try {
+          SetPrototypeDelete(httpConn.managedResources, requestRid);
+          core.close(requestRid);
+        } catch { /* pass */ }
```

Comment:
I don't really understand why it's inside a swallow-everything try/catch - seems like that could hide genuine bugs - but it's a pattern that's used elsewhere in this file so :shrug:

Reply:
`core.close()` should only throw `BadResource` when the thing has already been closed.